usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -11,7 +11,7 @@
 logger = logging.getLogger("console")
 
 DOWNLOAD_QUERY_SIZE = settings.MAX_DOWNLOAD_LIMIT
-KEYWORD_DATATYPE_FIELDS = ["{}.raw".format(i) for i in KEYWORD_DATATYPE_FIELDS]
+KEYWORD_DATATYPE_FIELDS = [field for field in KEYWORD_DATATYPE_FIELDS]
```

Comment:
why not just `list(KEYWORD_DATATYPE_FIELDS`?

Reply:
Good call. I will make the update.